CONTRIBUTING.md

```diff
@@ -22,3 +22,23 @@ If you are unsure about git and GitHub, and how to make your contribution,
 these [Git and GitHub for Poets tutorials](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) might help.
 
 Happy Coding!
+
+## Website Development
```

Comment:
I've written a quick [guide](https://github.com/CodingTrain/website/wiki/Core-Contribution-Guide) on how to set up a local dev environment. Maybe we should add a link to it here?

Reply:
Yes, of course, sorry didn't realize that the jekyll instructions were there also, this is redundant. I'll adapt!